tests/test_claude_wrapper_hooks.py

```
@@ -1422,6 +1426,30 @@ def test_live_socket_preserves_only_listed_claude_auth_keys(failures: list[str])
+    # Regression: a user with a custom ANTHROPIC_BASE_URL + non-Anthropic model slug
+    # (BYOK gateway / OpenAI-compatible proxy / internal LLM endpoint) must keep
+    # ANTHROPIC_MODEL across the wrapper, otherwise `claude --resume` reports
+    # "Session model <slug> could not be restored ... using the default model
+    # instead" and the session exits when a later turn re-resolves the model.
+    inherited = {
+        "ANTHROPIC_BASE_URL": "https://my-gateway.example.test/anthropic",
+        "ANTHROPIC_AUTH_TOKEN": "third-party-token",
+        "ANTHROPIC_MODEL": "glm-5.2[1m]",
+        "ANTHROPIC_DEFAULT_SONNET_MODEL": "glm-5.2[1m]",
+    }
+    code, auth_env, real_argv, stderr = run_wrapper_auth_env(
```

Ruff (0.15.18)

[warning] Unpacked variable `real_argv` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)